plugins/in_syslog/syslog_server.c

```diff
@@ -123,6 +126,7 @@ static int syslog_server_unix_create(struct flb_syslog *ctx)
 
     return 0;
 }
+#endif
```

Comment:
This conditional is fine but I need you to add this before the `#endif` : 

```
#else
static int syslog_server_unix_create(struct flb_syslog *ctx)
{
    return -1;
}
```

Reply:
I like your suggestion. I applied that change.